README.md

```diff
@@ -46,6 +46,10 @@ And this is what gets rendered (plus the CSS and Javascript you've specified):
 
 ## Release notes
 
+🚨📢 **Version 0.83** Autodiscovery module resolution changed. Following undocumented behavior was removed:
+- Previously, autodiscovery also imported any `[app]/components.py` files, and used `SETTINGS_MODULE` to search for component dirs.
```

Comment:
If you have your components in a components.py file, what do you need to do to keep using the project? I think that's a good question to answer here.

Reply:
Updated!